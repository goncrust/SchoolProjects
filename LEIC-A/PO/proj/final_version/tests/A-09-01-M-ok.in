4
3
969000
0
0
