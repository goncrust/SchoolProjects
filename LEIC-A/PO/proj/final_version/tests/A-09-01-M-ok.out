MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: Número do terminal: Operações de terminal: Operação inválida: O terminal '969000' não existe.
GESTÃO DE TERMINAIS
1 - Mostrar todos os terminais
2 - Registar terminal
3 - Operações de terminal
0 - Sair
Escolha uma opção: MENU PRINCIPAL
1 - Abrir
2 - Guardar
3 - Gestão de clientes
4 - Gestão de terminais
5 - Menu de consultas
6 - Ver saldo global
0 - Sair
Escolha uma opção: 